exchange/wrappers/luno.js

```diff
@@ -243,54 +312,55 @@ Trader.prototype.checkOrder = function(order, callback) {
 }
 
 Trader.prototype.cancelOrder = function(order, callback) {
+  log(name, 'cancelOrder() order id:', order);
+
   if (!order) {
     return callback('invalid order_id');
   }
   const process = (err, data) => {
     if (err) {
       if (_.includes(err.message, 'Cannot stop unknown')) {
-        console.log(name, 'unable to cancel order:', order, '(' + err.message + ') assuming success...');
-        // return callback(undefined, true);
+        log(name, 'unable to cancel order:', order, '(' + err.message + ') assuming success...');
       } else {
-        console.log(name, 'unable to cancel order:', order, '(' + err.message + ') aborting...');
+        log(name, 'unable to cancel order:', order, '(' + err.message + ') aborting...');
         return callback(err);
       }
     }
 
     if (data && !data.success) {
-      console.log('cancelOrder() --> status:', data.success);
+      log('cancelOrder() --> status:', data.success);
       return callback(undefined, false);
     }
 
     this.checkOrder(order, (error, orderStatus) => {
       if (error) {
-        console.log(name, 'cancelOrder\'s checkOrder failed. What do i do here?');
+        log(name, 'cancelOrder\'s checkOrder failed. What do i do here?');
```

Comment:
Good question.. At that point we are sure that the order was cancelled but we do not have fill information. Let's keep it like this: we bubble up the error and have Gekko figure out how to proceed.

Assuming it was or was not partially filled will probably lead to nasty bugs down the line.